### ISSUE TITLE
Wrong ports set for faasnetesd

### ISSUE THREAD

**Issue by alexellis** [openfaas/faas-netes#251]
The faas-netes ports need changing to 8081 (there is an error where they are set to 8080)

https://github.com/openfaas/faas-netes/blob/master/chart/openfaas/templates/faasnetesd-external-svc.yaml#L20

https://github.com/openfaas/faas-netes/blob/master/chart/openfaas/templates/faasnetesd-svc.yaml

https://github.com/openfaas/faas-netes/blob/master/yaml/faasnetesd-svc.yml#L13

This covers the helm chart and regular YAML.

`make charts` will also be needed and a bump to the chart version.
Alex

**Comment by mrtind** [comment]
I can work on this one

**Comment by alexellis** [comment]
Thanks Martin.

Please work with @viveksyngh if you have any issues, questions or queries. See the PR he just finished over at #252 

Alex

**Comment by alexellis** [comment]
Would like to see this in by EOD if possible because it's preventing faas-idler from working as expected. Does that work for you both?

**Comment by mrtind** [comment]
Yeah. I already did that I'm about to make pull request but i saw that the version is already 1.1.5. Should i make it 1.1.6 or?

**Comment by rawkode** [comment]
Issue closed by #254

**Comment by alexellis** [comment]
Derek: close
